README.md

```diff
@@ -115,3 +115,7 @@ by ```ProtectedGuard``` and won't be authenticated
 * ```AUTH_SERVICE``` - Authentication service token providers
 
 3. Provide your ```AuthenticationModule``` in your ```AppModule```
+
+### Customizing authentication headers
+
+By default, requests are intercepted and a ```{ Authorization: 'Bearer ${token}'}``` header is injected. To customize this behavior, implement the ```getHeaders``` method on your ```AuthenticationService```
```

Comment:
could you method interface to the class interface above?

Reply:
Decided not to do so: don't want to give the idea that this is mandatory to implement, I think it's really "advanced configuration"

Reply:
ok, sounds reasonable